mingw/bootstrap_vcpkg_cmake.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+OS_NAME="mingw"
```

Comment:
'mingw' is not an OS, maybe better to rename 'OS_NAME' to 'TARGET_PLATFORM_NAME' or smth similar?

Reply:
plt_name

Reply:
`PLATFORM_NAME`

Reply:
done.

Reply:
You changed this in ci_configure_libs_cmake.sh but not in this file. Please fix it here also

Reply:
done.